src/platform/testing/e2e/cypress/support/index.js

```diff
@@ -4,8 +4,33 @@ import 'cypress-plugin-tab';
 import 'cypress-real-events/support';
 import '@cypress/code-coverage/support';
 import addContext from 'mochawesome/addContext';
+import { configure } from '@testing-library/cypress';
 import './commands';
 
+// workaround for 'AssertionError: Timed out retrying after 4000ms: Invalid string length'
+// https://github.com/testing-library/cypress-testing-library/issues/241
+configure({
+  getElementError(message, container) {
+    const error = new Error(
+      [message, container.tagName].filter(Boolean).join('\n\n'),
+    );
+    error.name = 'TestingLibraryElementError';
+    return error;
+  },
+});
+
+before(() => {
+  cy.configureCypressTestingLibrary({
+    getElementError(message, container) {
+      const error = new Error(
+        [message, container.tagName].filter(Boolean).join('\n\n'),
+      );
+      error.name = 'TestingLibraryElementError';
+      return error;
+    },
+  });
+});
+
```

Comment:
I'm not sure that both of these are needed as it looks like they do the same thing. Since I'm the one that added these, I will look more into it and decide on which is better to resolve the string length failure issues we were seeing in ci/cd. 

The good news is that this did appear to solve those issues AFAIK 🎉

Reply:
Thanks for addressing that! Part of maintaining a useful suite of E2E tests is culling excess verifications and flows from time to time.

Reply:
Resolved this by removing the first configure() call. Turns out that is what is used inside `cy.configureCypressTestingLibrary` as shown in the source code here https://github.com/testing-library/cypress-testing-library/blob/main/src/add-commands.js#L7

🚀  hoping this helps prevent these failures in the ci/cd but if they resurface, then we can revisit.